src/traccuracy/track_errors/_ctc.py

```diff
@@ -69,13 +78,21 @@ def get_edge_errors(matched_data: Matched) -> None:
     gt_graph = matched_data.gt_graph
     node_mapping = matched_data.mapping
 
+    if comp_graph.edge_errors + gt_graph.edge_errors == 1:
+        graph_with_errors = "pred graph" if comp_graph.edge_errors else "GT graph"
+        raise ValueError(
+            f"Only {graph_with_errors} has edge errors annotated. "
+            "Please ensure either both or neither "
+            + "of the graphs have traccuracy annotations before running metrics."
+        )
+
     if comp_graph.edge_errors and gt_graph.edge_errors:
-        logger.info("Edge errors already calculated. Skipping graph annotation")
+        warnings.warn("Edge errors already calculated. Skipping graph annotation", stacklevel=2)
         return
 
     # Node errors must already be annotated
     if not comp_graph.node_errors and not gt_graph.node_errors:
-        logger.warning("Node errors have not been annotated. Running node annotation.")
+        logger.info("Node errors have not been annotated. Running node annotation.", stacklevel=2)
```

Comment:
In theory we could delete these conditionals and just let the check be handled by the node and edge error functions. But it doesn't bother me that much if we leave it.

Reply:
Discussed and decided to punt on this one. There are a couple of tests that do check this was logged, so we can always come back to it later.